### PR TITLE
Create Controller_Pinout

### PR DESCRIPTION
**Pull request by sukoshi1507**
Regarding reverse engineering you started, the hardware pinout is very useful.

### CODE REVIEW

File: Controller_Pinout [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+GND - common 
+P - power from the battery, a high impedance output. 
+Data - Serial interface 
++ 5V - rises after the charge and feeds into the steering wheel. 
+The signal P turns on the scooter. 
+
+5 - 3 Pin - interface to the BMS 
+signals: 
+the RX, the TX - serial interface. 
+L - line on the LED stop signal bufferezirovana mosfetom.
```
Comment (lukaville) [inline Controller_Pinout]: seems like a translation mistake :)

**Comment by lukaville** [comment]
Hi. Thanks for PR! Please, use Markdown formatting for this document (add `.md` extension and use [markdown syntax](https://guides.github.com/features/mastering-markdown/)).

**Comment by sukoshi1507** [comment]
Well, i don't speak russian so i had to rely on Google Translation ;)

i'll edit the file, spasibo!

**Comment by sukoshi1507** [comment]
Should be good now :)

**Comment by lukaville** [comment]
Ok, thanks again!

**Pull request merged**